packages/liotad/liotapkg.sh

```diff
@@ -33,19 +33,18 @@
 
 liota_config="/etc/liota/liota.conf"
 package_messenger_pipe=""
-package_response_pipe=""
 
 if ! [ `ps -ef | grep "liotad.py" | grep -v grep | wc -l` -gt 0 ]
 then
         echo "Liota package manager is not running. Please run it first!"
-        exit 1
+        exit -1
```

Comment:
What is the significance of changing it to "-1"????

I think they should be positive integers as they are independent of the rollback.

Reply:
You're right. I missed it in my review. All exit values should be >= 0.

Reply:
This is a revert of a commit. I don't want to introduce changes in a revert. This is what happens when we address more than one issue in a single commit/merge :-)

I validated that ICE client is not consuming these values. It is still a bug in liota. Please raise a separate bug for this. Also, exit values 1 and 2 have special meaning, so we should not use them in the return status. Hence, with or without this revert, the bug still needs to be raised.

Reply:
This is a new commit which is getting introduced and shouldn't change the fixed issue of exit status under the revert tag. It can be handled in separate commit but the correct code shouldn't be affected due to this commit. 

What is the significance of exit values "1" and "2" and "-1" and "-2"?

Reply:
Exit code 1 and 2 have special meanings: http://tldp.org/LDP/abs/html/exitcodes.html

As discussed, this is a git revert commit, and we should not introduce new changes.

Reply:
That page is describing what bash itself exits with; we can define whatever exit codes we want for when our scripts exit.